hangman-es6/vic-hangman-es6/scripts/main.js

```diff
@@ -0,0 +1,62 @@
+let gameHangmanPredator
+let $formInGame
```

Comment:
USE const instead, in case let declarations are only initialised once.

Reply:
hey @manuelbarzi  I've done all changes, but in this case, i can't put `const gameHangmanPredator` because, if I do that, i can't instance a `new HangPredator`.
As you can see in lane `16` from function **whenExecInitSubmit**.